cudaaligner/README.md

```diff
@@ -0,0 +1,15 @@
+# cudaaligner
+
+The `cudaaligner` package provides GPU-accelerated global alignment. Features include:
```

Comment:
Features include seems incomplete.

Reply:
Yes this was misleading. The `:` was for the `Library` and `Samples` section. But removed it to make it clearer.